### PR TITLE
Remove indent under your Collections.

### LINKED ISSUE #1632: Remove indent under "Your collections"
I think this is currently on `ul.collections`

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1632

## Why was this change made?
Fix indent.

## How was this change tested?
Local

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:


After:

File: app/packs/stylesheets/dashboard.scss [hunk 1]
```diff
@@ -38,4 +38,5 @@
 
 ul.collections {
   list-style-type: none;
+  padding-left: 0rem;
```
Comment (jcoyne) [inline app/packs/stylesheets/dashboard.scss:41]: I believe the convention is to remove the units for a "0" value. See https://github.com/CSSLint/csslint/wiki/Disallow-units-for-zero-values

Comment (justinlittman) [reply app/packs/stylesheets/dashboard.scss]: Ahhh ... fixed.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**